src/main/java/teammates/ui/controller/InstructorCourseInstructorEditSaveAction.java

```diff
@@ -55,6 +73,17 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         return result;
     }
 
+    protected int getNumberOfInstructorsDisplayedToStudents(String courseId) {
+        List<InstructorAttributes> instructors = logic.getInstructorsForCourse(courseId);
+        int numberOfInstructorsDisplayed = 0;
+        for (InstructorAttributes instructor : instructors) {
+            if (instructor.isDisplayedToStudents) {
+                numberOfInstructorsDisplayed++;
+            }
+        }
```

Comment:
This is fine but there is a better way to count matching items in a list using Java 8 streams. https://www.concretepage.com/java/jdk-8/java-8-stream-concat-count-sorted-distinct-example